response_test.go

```diff
@@ -119,3 +119,24 @@ func TestInstalledApplicationListResponse(t *testing.T) {
 
 	fmt.Printf("%v\n", response)
 }
+
+func TestProfileListResponse(t *testing.T) {
+	profileListResponseBody, err := ioutil.ReadFile("./test/responses/profilelist.plist")
+	if err != nil {
+		t.Fatal(err)
+	}
+	response := &Response{}
+	if err := plist.Unmarshal(profileListResponseBody, response); err != nil {
+		t.Fatal(err)
+	}
+
+	if response.Status != "Acknowledged" {
+		t.Fatal("Response status was not `Acknowledged`.")
```

Comment:
You probably want to change the t.Fatal to `t.Errorf`. The `Fatal` should only be used if the rest of the test cannot run, like if Unmarshal failed.